backend/databasemodel/alembic/versions/68c9b9ed3e26/upgrade.sql

```diff
@@ -0,0 +1,55 @@
+ALTER TABLE kooste.museovirastoarcrest_rkykohteet DROP CONSTRAINT museovirastoarcrest_rkykohteet_pk;
```

Comment:
Despite the name of the field, ID does not seem to be unique. See this example from Tampere region:
![Screenshot from 2022-04-05 09-02-54](https://user-images.githubusercontent.com/55135974/161690394-7171b60d-ccb9-4eb6-ad4d-7bed42a91e13.png)

Thus, I recommend going back to using OBJECTID as primary key. I do not really know why I had used camel case earlier since the field name seems to be caps locked.